packages/http-server/src/serialize.ts

```diff
@@ -19,7 +19,7 @@ const serializers = [
     serializer: (data: unknown) => (typeof data === 'string' ? data : xmlSerializer.parse({ xml: data })),
   },
   {
-    regex: /text\/plain/,
+    regex: /text\/(plain|html)\b/,
```

Comment:
I think we should probably use `type-is` instead of an regex so we can safely forward all the text types

Reply:
Sure,  but I am unaware of type-is lib. Can you point me to the documentation link for it?

Reply:
It is

Reply:
It's already a dependency so it shouldn't be a problem to use it.

Reply:
Sorry, I am not sure what should `toString` method return? Because, I am not sure where it is being used. So should this work?

```js
regex: {
      test: (value: string) => !!typeIs.is(value, ['text/*']),
      toString: () => 'text/*',
    },
```

Reply:
Put whatever you want — it does not matter.

Reply:
Thanks, will do